awx_collection/plugins/doc_fragments/auth.py

```diff
@@ -32,7 +32,7 @@ class ModuleDocFragment(object):
     description:
     - The Tower OAuth token to use.
     - If value not set, will try environment variable C(TOWER_OAUTH_TOKEN) and then config files
-    type: str
+    type: raw
```

Comment:
if this type is raw, I want to see some information in the `description:` with direction to the user. When is it not a string, and what format should be expected in that case?

Reply:
Changed it to:
- tower_oauthtoken
        The Tower OAuth token to use.
        This value can be in one of two formats.
        A string which is the token itself. (i.e. bqV5txm97wqJqtkxlMkhQz0pKhRMMX)
        A dictionary structure as returned by the tower_token module.
        If value not set, will try environment variable `TOWER_OAUTH_TOKEN' and then config files

Reply:
👍  to that description

but for whatever reason, I'm not seeing that in the current diff

Reply:
@beeankha asked me to hold off on the push until she finished up a yolo run. This will be up later today.